src/client/remote/ui/fileSystem.ts

```diff
@@ -183,18 +188,11 @@ export class RemoteFileSystem implements IFileSystemProvider {
     }
     public async createNew(remotePath: Uri, type: 'file' | 'directory' | 'notebook'): Promise<Uri | undefined> {
         const contentManager = await this.getContentManager();
-        try {
-            const model = await contentManager.newUntitled({ type, path: remotePath.fsPath });
-            if (model) {
-                const uri = Uri.file(model.path).with({ scheme: this.scheme });
-                this._fireSoon(
-                    { type: FileChangeType.Created, uri },
-                    { uri: remotePath, type: FileChangeType.Changed }
-                );
-                return uri;
-            }
-        } finally {
-            contentManager.dispose();
+        const model = await contentManager.newUntitled({ type, path: remotePath.fsPath });
```

Comment:
Not sure why you removed the dispose?

Reply:
Don't want to keep creating that everytime.
Didn't see any point. Basically makes not difference whether i create or destroy.

I figured it was cleaner to re-use same object that trying to be smart & disposing everytime.
When disconnecting from that server, that object gets disposed.